.gitignore

```diff
@@ -72,3 +72,5 @@ player/static/mfr/*
 !.gitkeep
 
 .cache
+
+node_modules/
```

Comment:
`/node_modules/` is preferred if  you target the top level directory. `node_modules/` ignores every directory in the whole tree if match.

Reply:
Hold off please @cslzchen. I'm mid-release.